src/Controller/TaskController.php

```diff
@@ -7,18 +7,20 @@
 use Doctrine\Persistence\ManagerRegistry;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class TaskController extends AbstractController
 {
-    #[Route('/tasks', name: 'task_list')]
-    public function listAction(ManagerRegistry $doctrine)
+    #[Route('/tasks', name: 'app_task_list')]
+    public function listAction(ManagerRegistry $doctrine): Response
     {
         return $this->render('task/list.html.twig', ['tasks' => $doctrine->getRepository(Task::class)->findAll()]);
     }
 
-    #[Route('/tasks/create', name: 'task_create')]
-    public function createAction(ManagerRegistry $doctrine, Request $request)
+    #[Route('/tasks/create', name: 'app_task_create')]
+    public function createAction(ManagerRegistry $doctrine, Request $request): RedirectResponse|Response
```

Comment:
injecter ManagerRegistry une seule fois dans le constructor

Reply:
Fixed => See commit [Issue #4 Implementation of automated tests](https://github.com/l-lemaitre/todolist/pull/9/commits/cddb13e1ca46e6ffbaba7587e68c926c7aedbaa7)